### PR TITLE
User/antony/#74

### PR DESCRIPTION
**Pull request by antonyanuraj**

### CODE REVIEW

File: bookwyrm_backend/src/main/java/com/bookwyrm/backend/review/input/ReviewUploadInput.java [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+import java.util.List;
+
 public class ReviewUploadInput {
 
     private String id;
+    private String title;
     private String author;
-    private String description;
+    private String content;
+    private List<Comment> commentList;
```
Comment (LukeBMorrow) [inline bookwyrm_backend/src/main/java/com/bookwyrm/backend/review/input/ReviewUploadInput.java:13]: Apologies, I believe I misunderstood when suggesting this. Reviews should never be created with pre-existing reviews, so we can leave the comment list out for the input.

**Review by LukeBMorrow — COMMENTED** [review]
Everything else looks good, but the comment object can be removed. Sorry again for the confusion.

File: bookwyrm_backend/src/main/java/com/bookwyrm/backend/review/input/ReviewUploadInput.java (hunk 1 above)
Comment (antonyanuraj) [reply bookwyrm_backend/src/main/java/com/bookwyrm/backend/review/input/ReviewUploadInput.java]: No problem, I have removed the comment object file and also removed commentList from the review endpoint files.

**Review by LukeBMorrow — APPROVED** [review]
Looks great!

**Pull request merged**